libs/common/src/sql-db/migrations/1706118619447-AddUtilisationReportTable.ts

```diff
@@ -0,0 +1,28 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddUtilisationReportTable1706118619447 implements MigrationInterface {
+  name = 'AddUtilisationReportTable1706118619447';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+      CREATE TABLE "UtilisationReport" (
+        "id" int NOT NULL IDENTITY(1, 1),
+        "bankId" nvarchar(255) NOT NULL,
+        "reportPeriodStartMonth" int NOT NULL,
+        "reportPeriodStartYear" int NOT NULL,
+        "reportPeriodEndMonth" int NOT NULL,
+        "reportPeriodEndYear" int NOT NULL,
+        "status" nvarchar(255) NOT NULL,
```

Comment:
postgres has enums but looks like MSSQL might not, sad :(

Reply:
Yeah, unfortunately not 😢